src/js/angular/repositories/ontop-repo.directive.js

```diff
@@ -75,12 +75,33 @@ function ontopRepoDirective($uibModal, RepositoriesRestService, toastr, Upload,
         };
 
         $scope.selectDriverByType = function (driverType) {
+            $scope.constructCustomFiledForDriver(driverType);
+
             $scope.copyDriverProperties($scope.supportedDriversData
                 .find((driver) => driver.driverType === driverType));
             // Call concatURL with proper labelName to apply changes to url field
             $scope.concatURL('hostName', $scope);
         };
 
+        $scope.constructCustomFiledForDriver = function (driver) {
+            switch (driver) {
+                case 'snowflake':
+                    $scope.supportedDriverLabels.hostName = 'Account identifier';
+                    $scope.supportedDriverLabels.databaseName = 'Warehouse';
+                    break;
+                case 'databricks':
+                    $scope.supportedDriverLabels.databaseName = 'HttpPath';
```

Comment:
The hostname for `dratabricks` and `dremio` will also be called `Account identifier` if `snowflake` is selected first.

Is this ok?

Reply:
It must be callled 'Hostname' for all other data sources, fixed in commit: 710a16d366ede3992d3931b4a6d3c4bd56b00933